### PR TITLE
Bounding box coordinates 

### PR DESCRIPTION
**Pull request by jhung0**

### CODE REVIEW

**Comment by karhohs** [comment]
This would solve #3571

File: cellprofiler/modules/measureobjectsizeshape.py [hunk 1]
```diff
@@ -112,6 +112,8 @@
    angles). The minimum and maximum Feret diameters are the smallest and
    largest possible diameters, rotating the calipers along all possible
    angles.
+-  *BoundingBoxMinX, BoundingBoxMinY, BoundingBoxMaxX, BoundingBoxMaxY:* *(2D only)* The
+   bounding box coordinates for an object. X corresponds to rows and Y corresponds to columns.
```
Comment (bethac07) [inline cellprofiler/modules/measureobjectsizeshape.py]: "X corresponds to rows and Y corresponds to columns."- this is vague.  Are you saying it's opposite to what we'd expect (ie X corresponds to position up-and-down and Y corresponds to position left-and-right)?  If so, we should change how the variables are assigned to correspond to the "expected" definitions of X and Y.  If not, we should clarify or omit this sentence.

**Comment by 0x00b1** [comment]
Bump @jhung0

File: cellprofiler/modules/measureobjectsizeshape.py (hunk 1 above)
Comment (bethac07) [reply cellprofiler/modules/measureobjectsizeshape.py]: Confirmed that the coordinates are currently backwards relative to meanings of X and Y elsewhere in CP

File: cellprofiler/modules/measureobjectsizeshape.py [hunk 2]
```diff
@@ -367,6 +373,19 @@ def run_on_objects(self, object_name, workspace):
...
                 nobjects = numpy.max(objects.indices)
+
+            bounding_box_min_x = []
+            bounding_box_min_y = []
+            bounding_box_max_x = []
+            bounding_box_max_y = []
+
+            for label, index in objects.get_labels():
+                props = skimage.measure.regionprops(label)
+                bounding_box_min_x += [int(prop.bbox[0]) for prop in props]
```
Comment (bethac07) [inline cellprofiler/modules/measureobjectsizeshape.py]: I think all you need to do here is set bounding_box_min_x to bbox[1], min_y to bbox[0], max_x to bbox[3], and max_y to bbox[2]

**Pull request closed without merging**